COPYING.txt

```diff
@@ -0,0 +1,177 @@
+
+                                 Apache License
```

Comment:
The official convention used on github would be to have a License file in the root of the repo. That's the reason I deleted this file

Reply:
Not sure what you mean. Isn't it in the root?

Reply:
Yep but it doesn't follow the file name convention people would look for the find the license of the code right?

Reply:
See for example this newly created repo. All done by using Github defaults https://github.com/danielmarbach/cautious-guacamole/blob/main/LICENSE

Reply:
> Yep but it doesn't follow the file name convention people would look for the find the license of the code right?

Ah I see. Well `COPYING.txt` has been working for me for decades and I am [using a convention older than GitHub](https://www.gnu.org/licenses/gpl-howto.en.html). 😄 In fact, GitHub uses a the [Licensee Ruby gem](https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/licensing-a-repository#detecting-a-license) that's [pretty smart about a number of conventions used out in the wild](https://github.com/licensee/licensee/blob/741eb5015f7027bb9de64593f3dcd12199c6d948/spec/licensee/project_files/license_file_spec.rb#L60-L86) (although `LICENSE` scores highest). On the repo's web front-end, the license file is recognised and called out:

![image](https://user-images.githubusercontent.com/20511/99832911-949be980-2b61-11eb-8fb7-9453421ffad6.png)

My original point about removing the license file was just for the package, but then I realised only later that it had been completely deleted.